client/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+		    <meta name="viewport" content="width=device-width">
```

Comment:
extra spaces. please add .editorconfig. Smth like that:
root = true
[*]
charset = utf-8
indent_style = space
indent_size = 2
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true

[*.md]
max_line_length = 0
trim_trailing_whitespace = false